js/AureliaJS.md

```diff
@@ -251,8 +251,8 @@ gulp watch
 
 * Browse to [`http://localhost:9000`](http://localhost:9000)  to see the application. You can make changes in the code found under `src` folder and the browser should auto-refresh itself while you save files. The Grid component is rendered as like the below screenshot.
 
-![](/js/Aurelia_images/skeleton_navigation.png)
+![Getting Started with JavaScript Skeleton](Aurelia_images/javascript-skeleton-navigation.png)
```

Comment:
Control name is wrong and update proper alt text.

Reply:
done